assets/airlines/fastga.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "FastGA",
-  "icao": "N",
+  "icao": "n",
```

Comment:
I worry about multiple airlines with the same `icao`.  We will be addressing this with #164 but this could cause problems.